lib/api/controllers/funnelController.js

```diff
@@ -398,6 +415,32 @@ class FunnelController {
       });
   }
 
+  /**
+   * @param {KuzzleRequest} request
+   * @param {String} prefix
+   * @return {KuzzleRequest}
+   */
+  performDocumentAlias(request, prefix = 'before') {
+    const { controller: controllerName, action: actionName } = request.input;
```

Comment:
(Nit) It's shorter like this
```suggestion
    const { controller, action } = request.input;
```